source/Userdocs/Software/jLEMS.md

```diff
@@ -1 +1,42 @@
+(jlems)=
 # jLEMS
+
+Java Interpreter for the Low Entropy Model Specification language. See http://lems.github.com/LEMS for the LEMS specification.
+
+```{admonition} jLEMS is the reference implementation of LEMS
+jLEMS was developed by Robert Cannon when the LEMS language was being devised and serves at the key reference for how to implement/interpret the language.
+```
+
+## Quick start
+
+Clone this repository:
```

Comment:
I think we need to provide a release zip that includes the jar and the runner script.

Since this is end-user documentation, I wouldn't put anything that requires building from source as a quick start. We don't want users to build stuff from source: they shouldn't need to know what maven is, and clone the repo etc. This is all.